syncer/streamer_controller.go

```diff
@@ -240,7 +243,11 @@ func (c *StreamerController) GetEvent(tctx *tcontext.Context) (event *replicatio
 	case *replication.RotateEvent:
 		// if is local binlog, binlog's name contain uuid information, need to save it
 		// if is remote binlog, need to add uuid information in binlog's name
-		if !c.setUUIDIfExists(string(ev.NextLogName)) {
+		c.Lock()
+		containUUID := c.setUUIDIfExists(string(ev.NextLogName))
```

Comment:
Do we need to protect the read of `c.uuidSuffix` bellow?

Reply:
yes, fix in fef9390